test/font/jasmine-boot.js

```diff
@@ -76,9 +76,9 @@ function initializePDFJS(callback) {
     },
   });
 
-  var catchingExceptions = queryString.getParam('catch');
-  env.catchExceptions(typeof catchingExceptions === 'undefined' ?
-                      true : catchingExceptions);
+  var stoppingOnSpecFailure = queryString.getParam('failFast');
+  env.stopOnSpecFailure(typeof stoppingOnSpecFailure === 'undefined' ?
+                        true : stoppingOnSpecFailure);
```

Comment:
From a cursory look at https://github.com/jasmine/jasmine/commit/e15f273f06c6b8c7902050fe500486c056802ed1, which necessitated this change, shouldn't the default now be `false` instead here?